src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -492,7 +492,7 @@ public static class Parser
         private static readonly Regex FileExtensionRegex = new Regex(@"\.[a-z0-9]{2,4}$",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly RegexReplace SimpleTitleRegex = new RegexReplace(@"(?:(480|540|576|720|1080|2160)[ip]|[xh][\W_]?26[45]|DD\W?5\W1|[<>?*]|848x480|1280x720|1920x1080|3840x2160|4096x2160|(8|10)b(it)?|10-bit)\s*?",
```

Comment:
This was causing any hash value with either `8B` or `10B` to be removed when the title is simplified. For example `[4228bb8c]` would become `[422b8c]` and then failed the hash regex because it's not 8 characters.

The updated expression should still match `8b`, `8bit`, `10b`, `10bit`, or `10-bit` for removal, but `8b` and `10b` can't have an immediate word character after it. I didn't use `\w` because of the underscore and `_8b_` could be what people use in their files.

Reply:
There's still cases where the hash won't match, but I think a decision would need to be made.
Hashes that end with `10b` or `8b` still might not match e.g. `[15AF298B]` or `[15AF210B]`

Reply:
An alternative could be to check if the `<hash>` match has content, and if not, do an explicit RegEx against the raw release title trying to find the hash.

Pros:
- Don't have to keep tweaking existing expressions for niche cases of Anime naming
- Could find hashes in non-anime releases if they had them (I don't know of any off the top of my head)

Con:
- Increases processing time with an additional check. The check is only perform when one of the title matches hits, so it's not a 2x across the board. It would be a simple regex `(?<hash>[(\[][0-9a-f]{8}[)\]])$`, so should be relatively quick.
- Could find hashes in non-anime releases (leading to questions of "why does this one have a hash but that one doesn't?")

Reply:
I think you should be able to do a lookbehind and only match if it's proceeded by a space, underscore, or period, or a negative look behind to make sure it's not proceeded by a valid character in the hash.